src/common/Navigation/styled.js

```diff
@@ -1,29 +1,125 @@
 import { NavLink } from "react-router-dom";
 import styled from "styled-components";
+import { ReactComponent as CameraIcon } from "../../images/camera.svg"
+import { ReactComponent as MagnifyingGlass } from "../../images/search.svg"
+import img from "../../images/search.svg";
 
 export const NavbarContainer = styled.nav`
-    height: 94px;
+    max-height: 94px;
     display: flex;
-    justify-content: center;
-    background-color: ${({theme}) => theme.color.black};
+    justify-content: space-evenly;
+    background-color: ${({ theme }) => theme.color.black};
+
+    @media(max-width: 860px) {
+        flex-wrap: wrap;
+        max-height: 144px;
+    }
+`;
+
+export const StyledNavbar = styled.ul`
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    padding-right: 7vw;
+
+    @media(max-width: 860px) {
+        padding: 0px;
+    }
 `;
 
 export const StyledNavLink = styled(NavLink)`
     padding: 8px 24px;
```

Comment:
I think it should be `padding: 13.5px 24px;`. Something is wrong on Figma, check it out.

Reply:
You need to show me what's up, I couldn't find it:/